README.md

```diff
@@ -1,5 +1,167 @@
+	<br> We also have a set of tests for this module and they are kept inside the <i>tests</i> package.
+
+## Data structure
+For an automated pipeline run, the patient data is divided in three categories:
+- raw data
+    <br> Usually scans, that are used as input files for the pipeline (e.g. T1, DWI, CT).
+- configurations
+    <br> This is a folder with configuration files. Each configuration file is described in the <b>How to launch</b> section. 
+    <br> This folder is patient specific. All the files should be filled in by the user or generated automatically with the <b>run_sequential.py</b> described also in <b>How to launch</b> section.
+- output data
+    <br> These are the files generated by the pipeline. The output can be structured using the configuration file <b>rc_out.txt</b>.
+
+For a multi-patient sequential run, the data needs to be structured in a similar manner for each patient. Also, it is important to name the patient folders in a predefined manner. As an example, a simple folder structure can be:
+- data_folder
+    - raw_data
+        - patient1
+        - patient2
+        - patient3
+    - configurations
+        - patient1
+        - patient2
+        - patient3
+    - outputs
+        - patient1
+        - patient2
+        - patient3
+        
+
+## How to launch
+
+### Entry point:
+There are 2 available entry points for the pipeline. They are both under <i>pegasus</i> folder. In order to use these entry points, there are, in both cases, some configurations to be defined first. These configurations are kept as a folder specific to each patient and are explained in the next section.
+
+The pipeline can be started using one of the following entry points:
+- <b>main_pegasus.sh</b>
+    <br> This is the most straight-forward one. It starts one pipeline run for a single patient based on a set of predefined configurations. Command to launch the pipeline with this script: 
+    <br><b><i>sh main_pegasus.sh path_to_configurations_folder path_to_dax_folder</i></b>
+    
+    The arguments:
+    - <i>path_to_configurations_folder</i> represents the path to the patient configuration files (e.g. data_folder/configurations) 
+    - <i>path_to_dax_folder</i> represents the folder where the dax will be generated (e.g. data_folder/configurations/dax)
+    
+    This entry point has the disadvantage that the user should manually fill in all the configuration files.
+
+- <b>run_sequential.py</b> 
+    <br> This is a little more complex. It is used to start pipeline runs for a list of patients with similar configurations. As the name is suggesting, the runs will be started sequentially.
+    <br> Command to launch the pipeline with this script: <b><i>python run_sequentially.py</i></b>
+    <br> This script does not need arguments, but it needs the user to define the necessary configurations inside it. The necessary configurations are described bellow:
```

Comment:
This script does not need arguments, but it needs the user to edit the necessary configurations inside file run_sequentially.py.